policies/experimental-features.md

```diff
@@ -0,0 +1,46 @@
+Experimental Feature Policy
+===========================
+
+A feature in OpenSSL may be designated as an experimental feature.
+
+No stability or compatibility guarantees are offered for experimental features.
+An experimental feature, or any API, protocol or file format which is part of an
+experimental feature, may be changed arbitrarily and in compatibility-breaking
+ways, or removed entirely at any time, including between releases.
+
+An experimental feature is not enabled by default and must be explicitly enabled
+at build time in order to be used. The enable flag to enable an experimental
+feature must have the form `enable-unstable-XXX`. The corresponding `ifdef`
+shall have the form `OPENSSL_NO_UNSTABLE_XXX`.
+
+A prominent warning should be issued by the configure script if any experimental
+feature is enabled, with a link to a webpage where people can learn more about
+experimental features.
+
+Experimental features may have associated documentation. This documentation is
+not built by default unless the corresponding experimental feature is enabled.
+Man pages for experimental features shall feature a prominent warning that the
+man page relates to an experimental feature.
+
+Experimental features exist to support the release engineering process where the
+use of a feature branch is not feasible or desirable. Feature branches should be
+used instead of experimental features where it is reasonable to do so.
+
+Experimental features are not required to be in a buildable or usable state at
+the time of a release or at the time of the creation of a release branch.
+
+Nothing relating to an experimental feature shall be considered a CVE or handled
+under the security process.
```

Comment:
The security policy should be updated in this case to make it clear that issues in experimental features are outside of the threat model